### PR TITLE
[DOCFIX] Update MinIO doc for 2.0

### PR DESCRIPTION
**Pull request by tieujason330**
Docs were straightforward. I had to follow MinIO's getting started page to get everything setup. 

- Changes are mainly small typos and changing 'Minio' to 'MinIO' to match their spelling

### CODE REVIEW

**Comment by tieujason330** [comment]
jenkins, test this please

**Comment by tieujason330** [comment]
jenkins, test this please

**Comment by tieujason330** [comment]
jenkins, test this please

**Comment by tieujason330** [comment]
jenkins, test this please

**Comment by tieujason330** [comment]
@ZacBlanco PTAL, thanks

File: docs/en/ufs/Minio.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+nickname: MinIO
 group: Under Stores
 priority: 10
 ---
 
 * Table of Contents
 {:toc}
 
-This guide describes how to configure Alluxio with [Minio](https://minio.io/) as the
+This guide describes how to configure Alluxio with [MinIO](https://minio.io/) as the
```
Comment (ZacBlanco) [inline docs/en/ufs/Minio.md]: I think their new website is `https://min.io/`?

File: docs/en/ufs/Minio.md [hunk 2]
```diff
@@ -1,17 +1,17 @@
 ---
 layout: global
-title: Minio
-nickname: Minio
+title: MinIO
```
Comment (ZacBlanco) [inline docs/en/ufs/Minio.md]: On the Minio product page they use `MinIO`. Is there a reason to change that here?

Comment (ZacBlanco) [reply docs/en/ufs/Minio.md]: https://min.io/product

File: docs/en/ufs/Minio.md (hunk 1 above)
Comment (tieujason330) [reply docs/en/ufs/Minio.md]: It redirects to min.io but I'll update the url

**Comment by tieujason330** [comment]
jenkins, test this please

**Comment by cheng-chang** [comment]
alluxio-bot, merge this please

**Comment by ZacBlanco** [comment]
Don't forget to cherry pick this

**Comment by calvinjia** [comment]
alluxio-bot, cherry-pick this to branch-2.0 please.

**Pull request merged**